ui/app/styles/components/console-ui-panel.scss

```diff
@@ -11,8 +11,8 @@
   min-height: 0;
   overflow: scroll;
   right: 0;
-  top: $header-height;
-  transition: min-height $speed ease-out, transform $speed ease-in;
+  top: $spacing-xxxl;
+  transition: min-height $speed $easing, transform $speed ease-in;
```

Comment:
Agreed. It was a variable, then I removed it, then I added it back in. I'll keep in for now, and get clarification on it alongside the sizing var conversation. Basically, I think we need a convo on when and what should be a var and the rules to follow when making one. 

Note: I ultimately kept it a var because I saw it was the standard for what used on this transition property. So I went ahead and replaced it where it could be used.